tutorials/migrating/upgrading_to_godot_4.rst

```diff
@@ -438,6 +442,30 @@ table to find its new name.
 - TileMap's ``world_to_map()`` is now ``local_to_map()``.
 - TileMap's ``map_to_world()`` is now ``map_to_local()``.
 - Transform2D's ``xform()`` is ``mat * vec`` and ``xform_inv()`` is ``vec * mat``.
+- XRPositionalTracker's ``_set_name()`` is now ``get_tracker_name()``.
+- BaseButton's ``_unhandled_input()`` is now ``_unhandled_key_input()``.
```

Comment:
What are you basing this on? Why is this not just `_unhandled_input`?

Reply:
https://github.com/godotengine/godot/pull/42100 It looks like originally the intention was to replace `_unhandled_input` with `_unhandled_key_input` but the whole system changed somewhere between 4.0 and 4.2. I'll add it to my list of things that need to be removed from the rename_map.